### ISSUE TITLE
Simple formatting in tables

### ISSUE THREAD

**Issue by luisfilipeoliveira** [hummingbot/hummingbot#4509]
## Why
The tables that appear in the output panel are unformatted for visual comfort at the time of analysis.

![Screenshot 2021-10-19 at 23.06.44.png](https://images.zenhubusercontent.com/57179e45bf93871ccd99c501/02537dd0-0a95-4043-a3f3-30029c70559f)

## What
If possible there is a simple format as this image represents. Simple character separations immediately create a contrast in the columns and rows we analyze.

![Screenshot 2021-10-19 at 23.05.11.png](https://images.zenhubusercontent.com/57179e45bf93871ccd99c501/4d4bd507-4b50-40d7-938b-4684d39e6a78)

## Definition of Done
Make changes to tables related to information data (with tabulate 0.8.9)

- [ ] `history`
- [ ] `connect`
- [ ] `config`
- [ ] `orderbook`
- [ ] `ticker`
- [ ] `trades`

**Comment by aarmoa** [comment]
Full documentation https://bitbucket.org/astanin/python-tabulate/src/master/

**Comment by aarmoa** [comment]
What is the extension of this task? Should we apply tabulate to all the tables that we print in the client? (remember that we have the status table defined in each strategy)

**Comment by dennisocana** [comment]
@aarmoa correct, my understanding is we should apply tabulate to all tables especially `history`, `connect`, `config`, and `status` command outputs.

**Comment by luisfilipeoliveira** [comment]
@aarmoa the example we have above in the design I used `psql` table format (_is like tables formatted by Postgres' psql cli_)

**Comment by petioptrv** [comment]
Addressed in https://github.com/CoinAlpha/hummingbot/pull/54

**Comment by nikspz** [comment]
Merged https://github.com/hummingbot/hummingbot/pull/5081
